ttnn/cpp/ttnn/operations/data_movement/reshape_view/reshape_pybind.cpp

```diff
@@ -0,0 +1,124 @@
+
+namespace ttnn::operations::data_movement {
+
+namespace detail {
+
+template <typename data_movement_operation_t>
+void bind_reshape_view(pybind11::module& module, const data_movement_operation_t& operation, const char* doc) {
+    bind_registered_operation(
+        module,
+        operation,
+        doc,
+        ttnn::pybind_overload_t{
+            [](const data_movement_operation_t& self,
+               const ttnn::Tensor& input_tensor,
+               const ttnn::Shape& new_shape
+               ) -> ttnn::Tensor {
+                return self(input_tensor, new_shape);
+            },
+            py::arg("input_tensor"),
+            py::arg("new_shape"),
+            },
+        ttnn::pybind_overload_t{
+            [](const data_movement_operation_t& self,
+               const ttnn::Tensor& input_tensor,
+               const std::array<int32_t, 1>& new_shape
+               ) -> ttnn::Tensor {
+                return self(input_tensor, ttnn::Shape(new_shape));
+            },
+            py::arg("input_tensor"),
+            py::arg("new_shape"),
+            },
+        ttnn::pybind_overload_t{
+            [](const data_movement_operation_t& self,
+               const ttnn::Tensor& input_tensor,
+               const std::array<int32_t, 2>& new_shape
+               ) -> ttnn::Tensor {
+                return self(input_tensor, ttnn::Shape(new_shape));
+            },
+            py::arg("input_tensor"),
+            py::arg("new_shape"),
+            },
+        ttnn::pybind_overload_t{
+            [](const data_movement_operation_t& self,
+               const ttnn::Tensor& input_tensor,
+               const std::array<int32_t, 3>& new_shape
+               ) -> ttnn::Tensor {
+                return self(input_tensor, ttnn::Shape(new_shape));
+            },
+            py::arg("input_tensor"),
+            py::arg("new_shape"),
+            },
+        ttnn::pybind_overload_t{
+            [](const data_movement_operation_t& self,
+               const ttnn::Tensor& input_tensor,
+               const std::array<int32_t, 4>& new_shape
+               ) -> ttnn::Tensor {
+                return self(input_tensor, ttnn::Shape(new_shape));
+            },
+            py::arg("input_tensor"),
+            py::arg("new_shape"),
+            },
+        ttnn::pybind_overload_t{
+            [](const data_movement_operation_t& self,
+               const ttnn::Tensor& input_tensor,
+               const std::array<int32_t, 5>& new_shape
+               ) -> ttnn::Tensor {
+                return self(input_tensor, ttnn::Shape(new_shape));
+            },
+            py::arg("input_tensor"),
+            py::arg("new_shape"),
+            },      
+
+        );
+}
+
+}  // namespace detail
+
+
+void py_bind_reshape_view(pybind11::module& module) {
+    detail::bind_reshape_view(
+        module,
+        ttnn::reshape,
+
+        R"doc(reshape(input_tensor: ttnn.Tensor, ttnn.Shape new_shape) -> ttnn.Tensor
```

Comment:
After https://github.com/tenstorrent/tt-metal/pull/12075 you're not supposed to have the header.

Reply:
Also, do not use `:attr:` directives, just take a look at other ops to see what the new docs format is.